cypress/support/pages/tasks/pageTasks.js

```diff
@@ -100,45 +166,173 @@ class Tasks {
     cy.get(Tasks.#visibilityDueDateInput).should('have.value', dueDateCheckValue)
   }
 
-  // compareVisibilityStartDate(visibilityStartDate, visibilityStartTime){
+  draftIsDisabled(){
```

Comment:
method-> verb-noun --> e.g  seeDisabledDraft

Reply:
changed to seeDraftIsDisabled